.github/workflows/test.yaml

```diff
@@ -47,3 +47,44 @@ jobs:
         with:
           enable-cache: true
           project-path: 'testdata'
+
+  test-action-with-sha256-checksum:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install devbox
+        uses: ./
+        with:
+          devbox-version: 0.5.5
+          enable-cli-cache: 'false'
```

Comment:
I wonder if it is clearer to do `refresh-cli: true`, or `disable-cli-cache: true` and set the default to `false`?

Reply:
Let's do `refresh-cli`, that sounds good to me.